data/json/itemgroups/SUS/domestic.json

```
@@ -615,14 +615,55 @@
     "//2": "This group is for a bathroom sink.",
     "subtype": "collection",
     "entries": [
+      { "item": "soap", "count": [ 1, 4 ], "prob": 75 },
       { "item": "soap_holder", "prob": 40 },
-      { "item": "soap", "count": [ 1, 4 ], "prob": 90 },
       { "item": "razor_shaving", "count": [ 1, 2 ], "prob": 20 },
+      { "item": "cotton_ball", "prob": 50 },
       { "item": "bleach", "prob": 15 },
       { "item": "ammonia", "prob": 20 },
+      { "item": "string_floss", "count": [ 1, 2 ], "prob": 60 },
       { "item": "toothbrush_plain", "count": [ 1, 3 ], "prob": 45 }
     ]
   },
+  {
+    "id": "SUS_bathroom_medicine",
+    "type": "item_group",
+    "//": "SUS item groups are collections that contain a reasonable realistic distribution of items that might spawn in a given storage furniture.",
+    "//2": "This group is for the medicine cabinet.",
+    "subtype": "collection",
+    "entries": [
+      {
+        "distribution": [
+          { "item": "aspirin", "prob": 60 },
+          { "item": "aspirin", "prob": 40 },
+          { "item": "codeine", "prob": 30 },
+          { "item": "tramadol", "prob": 10 }
+        ],
+        "prob": 50
```

```suggestion
        "prob": 95
```
I think a pain medication of some sort is something you should be almost guaranteed to find in a medicine cabinet.